src/NServiceBus.Transport.Bridge/MessageShovel.cs

```diff
@@ -37,13 +37,16 @@ public async Task TransferMessage(TransferContext transferContext, CancellationT
                 //We _do not_ transform the ReplyToAddress header
                 TransformAddressHeader(messageToSend, targetEndpointRegistry, FaultsHeaderKeys.FailedQ);
             }
-            else if (IsRetryMessage(messageToSend))
+            else if (IsAuditMessage(messageToSend))
             {
-                //This is a message retried from ServiceControl. Its ReplyToAddress header has been preserved (as stated above) so we don't need to transform it back
+                //This is a message sent to the audit queue. We _do not_ transform any headers
```

Comment:
```suggestion
                //This is a message sent to the audit queue. We _do not_ transform any headers. 
                //This check needs to be done _before_ the retry message check because we don't want to treat audited retry messages as retry messages.
```